src/components/DateTimePicker/Internal/OcPicker.tsx

```diff
@@ -83,18 +86,23 @@ function InnerPicker<DateType>(props: OcPickerProps<DateType>) {
     popupPlacement,
     popupStyle,
     shape = DatePickerShape.Rectangle,
+    showNow,
+    showOk,
```

Comment:
should you force this true if changeOnBlur is false?

Reply:
These are true by default despite `changeOnBlur` to avoid regressions. One must explicitly set them to false at implementation time.